### ISSUE TITLE
--report and --log anomolies

### ISSUE THREAD

**Issue by goffinf** [aquasecurity/kube-hunter#108]
**Behaviour:**

Depending on the options set either output appears or not.

**Configuration:**

Running as a K8s Job.

This **DOES** produce output ... note I have -**-log set to NONE and --report to JSON** ...

```
spec:
  containers:
  - args:
    - --remote
    - foo.bar.baz.com
    - --log
    - none
    - --report
    - json
    command:
    - python
    - kube-hunter.py
```

This does **NOT** produce output ... same as above .. just changed **--report to PLAIN**

```
spec:
  containers:
  - args:
    - --remote
    - foo.bar.baz.com
    - --log
    - none
    - --report
    - plain
    command:
    - python
    - kube-hunter.py
```

This **DOES** produce output ... just changed **--log to INFO**

```
spec:
  containers:
  - args:
    - --remote
    - foo.bar.baz.com
    - --log
    - info
    - --report
    - plain
    command:
    - python
    - kube-hunter.py
```

I haven't tried all combinations (although a few more than I have shown here) but from these very limited tests it suggests that if --log is set to a vault other than NONE, output is produced, or if --log is NONE then --report only works with JSON ??

**Comment by iYehuda** [comment]
The bug occurs when the log level is higher than INFO.
The reason for that is that the report is printed via `log.info` when in plain report type.

**Comment by iYehuda** [comment]
Resolved via #175
